sklearn/linear_model/bayes.py

```diff
@@ -216,10 +221,48 @@ def fit(self, X, y):
         self.alpha_ = alpha_
         self.lambda_ = lambda_
         self.coef_ = coef_
+        sigma_ = np.dot(Vh.T,
+                        Vh / (eigen_vals_ + lambda_ / alpha_)[:, None])
+        self.sigma_ = (1. / alpha_) * sigma_
 
         self._set_intercept(X_offset, y_offset, X_scale)
         return self
 
+    def predict(self, X, predict_std=False):
+        """Predict using the linear model. In addition to the mean of the
+        predictive distribution, also its standard deviation can be returned.
+
+        See: http://www.utstat.toronto.edu/~rsalakhu/sta4273/notes/Lecture2.pdf
+        Slide 15, titled "Predictive Distribution"
+        Russ's beta is our self.beta_
+        Russ's alpha is our self.lambda_
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape = (n_samples, n_features)
+            Samples.
+
+        predict_std : boolean, optional
```

Comment:
Please use `return_std` like `GaussianProcessRegressor`, and check its docstring for consistency.